pkg/ingester/ingester_v2.go

```diff
@@ -29,9 +30,18 @@ const (
 	errTSDBCreateIncompatibleState = "cannot create a new TSDB while the ingester is not in active state (current state: %s)"
 )
 
+// UserTSDB is a users TSDB
+type UserTSDB struct {
+	sync.Mutex
+
+	*tsdb.DB
+	cancel      context.CancelFunc
```

Comment:
Maybe naming it `shipperCancel` may be more clear to the reader?

Reply:
Sure. Changed.